suites/message/create_actor.go

```diff
@@ -109,15 +109,15 @@ func TestInitActorSequentialIDAddressCreate(t *testing.T, factory state.Factorie
 	var initialBal = abi_spec.NewTokenAmount(200_000_000_000)
 	var toSend = abi_spec.NewTokenAmount(10_000)
 
-	sender, senderID := td.NewAccountActor(drivers.SECP, initialBal)
+	sender, _ := td.NewAccountActor(drivers.SECP, initialBal)
 
 	receiver, receiverID := td.NewAccountActor(drivers.SECP, initialBal)
 
 	firstPaychAddr := utils.NewIDAddr(t, utils.IdFromAddress(receiverID)+1)
 	secondPaychAddr := utils.NewIDAddr(t, utils.IdFromAddress(receiverID)+2)
 
-	firstInitRet := td.ComputeInitActorExecReturn(senderID, 0, firstPaychAddr)
-	secondInitRet := td.ComputeInitActorExecReturn(senderID, 1, secondPaychAddr)
+	firstInitRet := td.ComputeInitActorExecReturn(sender, 1, firstPaychAddr)
+	secondInitRet := td.ComputeInitActorExecReturn(sender, 2, secondPaychAddr)
```

Comment:
Continue to pass the ID based address here since it represents the runtimes immediate caller and is always expected to be an ID-address: https://filecoin-project.github.io/specs/#vmruntime-interface

Reply:
Thanks! I thought the spec was ambiguous here and picked the pubkey address, but it is very much not ambiguous. Changes made!